### ISSUE TITLE
[Issue]: OSError: [WinError 126] Não foi possível encontrar o módulo especificado. Error loading "E:\ia\automatic\venv\lib\site-packages\torch\lib\fbgemm.dll" or one of its dependencies.

### ISSUE THREAD

**Issue by manotroll** [vladmandic/automatic#3375]
### Issue Description

I'm doing a clean install

### Version Platform Description

E:\ia\automatic>webui.bat --version
Using VENV: E:\ia\automatic\venv
21:47:50-957794 INFO     Starting SD.Next
21:47:50-961698 INFO     Logger: file="E:\ia\automatic\sdnext.log" level=INFO size=14942 mode=append
21:47:50-963649 INFO     Python version=3.10.11 platform=Windows bin="E:\ia\automatic\venv\Scripts\python.exe"
                         venv="E:\ia\automatic\venv"
21:47:51-087802 INFO     Version: app=sd.next version=unknown branch=unknown ui=unknown
21:47:51-102008 INFO     Platform: arch=AMD64 cpu=AMD64 Family 25 Model 33 Stepping 0, AuthenticAMD system=Windows
                         release=Windows-10-10.0.22631-SP0 python=3.10.11
21:47:51-105915 INFO     HF cache folder: C:\Users\manotroll\.cache\huggingface\hub
21:47:51-111806 INFO     Python version=3.10.11 platform=Windows bin="E:\ia\automatic\venv\Scripts\python.exe"
                         venv="E:\ia\automatic\venv"
21:47:51-116720 INFO     AMD ROCm toolkit detected
21:47:51-151042 WARNING  HIP SDK is detected, but no Torch release for Windows available
21:47:51-152993 INFO     For ZLUDA support specify '--use-zluda'
21:47:51-153970 INFO     Using CPU-only torch
21:47:51-171697 ERROR    Could not load torch: [WinError 126] Não foi possível encontrar o módulo especificado. Error
                         loading "E:\ia\automatic\venv\lib\site-packages\torch\lib\fbgemm.dll" or one of its
                         dependencies.

### Relevant log output

```shell
E:\ia\automatic>webui.bat --experimental --debug --upgrade --update
Using VENV: E:\ia\automatic\venv
21:43:02-486060 INFO     Starting SD.Next
21:43:02-489992 INFO     Logger: file="E:\ia\automatic\sdnext.log" level=DEBUG size=65 mode=create
21:43:02-491951 INFO     Python version=3.10.11 platform=Windows bin="E:\ia\automatic\venv\Scripts\python.exe"
                         venv="E:\ia\automatic\venv"
21:43:02-617036 INFO     Version: app=sd.next version=unknown branch=unknown ui=unknown
21:43:02-657731 ERROR    Error running git: None / rev-parse HEAD
21:43:02-658709 DEBUG    Git output: fatal: detected dubious ownership in repository at 'E:/ia/automatic'
                         'E:/ia/automatic' is owned by:
                                 (inconvertible) (S-1-5-21-291463465-3360491487-3224188617-1001)
                         but the current user is:
                                 DESKTOP-6RE3Q6G/manotroll (S-1-5-21-2930017012-368641518-4144343359-1001)
                         To add an exception for this directory, call:

                                 git config --global --add safe.directory E:/ia/automatic
21:43:02-781886 INFO     Updating main repository
21:43:02-821129 ERROR    Error running git: None / add .
21:43:02-823081 DEBUG    Git output: fatal: detected dubious ownership in repository at 'E:/ia/automatic'
                         'E:/ia/automatic' is owned by:
                                 (inconvertible) (S-1-5-21-291463465-3360491487-3224188617-1001)
                         but the current user is:
                                 DESKTOP-6RE3Q6G/manotroll (S-1-5-21-2930017012-368641518-4144343359-1001)
                         To add an exception for this directory, call:

                                 git config --global --add safe.directory E:/ia/automatic
21:43:02-867652 ERROR    Error running git: None / stash
21:43:02-868628 DEBUG    Git output: fatal: detected dubious ownership in repository at 'E:/ia/automatic'
                         'E:/ia/automatic' is owned by:
                                 (inconvertible) (S-1-5-21-291463465-3360491487-3224188617-1001)
                         but the current user is:
                                 DESKTOP-6RE3Q6G/manotroll (S-1-5-21-2930017012-368641518-4144343359-1001)
                         To add an exception for this directory, call:

                                 git config --global --add safe.directory E:/ia/automatic
21:43:02-914743 ERROR    Error running git: None / config rebase.Autostash true
21:43:02-915719 DEBUG    Git output: fatal: not in a git directory
21:43:02-954932 ERROR    Error running git: . / pull --rebase --force
21:43:02-956885 DEBUG    Git output: fatal: detected dubious ownership in repository at 'E:/ia/automatic'
                         'E:/ia/automatic' is owned by:
                                 (inconvertible) (S-1-5-21-291463465-3360491487-3224188617-1001)
                         but the current user is:
                                 DESKTOP-6RE3Q6G/manotroll (S-1-5-21-2930017012-368641518-4144343359-1001)
                         To add an exception for this directory, call:

                                 git config --global --add safe.directory E:/ia/automatic
21:43:03-003973 ERROR    Error running git: None / log -1 --pretty=format:"%h %ad"
21:43:03-005932 DEBUG    Git output: fatal: detected dubious ownership in repository at 'E:/ia/automatic'
                         'E:/ia/automatic' is owned by:
                                 (inconvertible) (S-1-5-21-291463465-3360491487-3224188617-1001)
                         but the current user is:
                                 DESKTOP-6RE3Q6G/manotroll (S-1-5-21-2930017012-368641518-4144343359-1001)
                         To add an exception for this directory, call:

                                 git config --global --add safe.directory E:/ia/automatic
21:43:03-012798 INFO     Upgraded to version: fatal: detected dubious ownership in repository at 'E:/ia/automatic'
                         'E:/ia/automatic' is owned by:
                                 (inconvertible) (S-1-5-21-291463465-3360491487-3224188617-1001)
                         but the current user is:
                                 DESKTOP-6RE3Q6G/manotroll (S-1-5-21-2930017012-368641518-4144343359-1001)
                         To add an exception for this directory, call:

                                 git config --global --add safe.directory E:/ia/automatic
21:43:03-033869 INFO     Platform: arch=AMD64 cpu=AMD64 Family 25 Model 33 Stepping 0, AuthenticAMD system=Windows
                         release=Windows-10-10.0.22631-SP0 python=3.10.11
21:43:03-037775 DEBUG    Setting environment tuning
21:43:03-038751 INFO     HF cache folder: C:\Users\manotroll\.cache\huggingface\hub
21:43:03-039755 DEBUG    Torch allocator: "garbage_collection_threshold:0.80,max_split_size_mb:512"
21:43:03-040736 DEBUG    Torch overrides: cuda=False rocm=False ipex=False diml=False openvino=False
21:43:03-042690 DEBUG    Torch allowed: cuda=True rocm=True ipex=True diml=True openvino=True
21:43:03-048549 INFO     Python version=3.10.11 platform=Windows bin="E:\ia\automatic\venv\Scripts\python.exe"
                         venv="E:\ia\automatic\venv"
21:43:03-054940 INFO     AMD ROCm toolkit detected
21:43:03-071128 DEBUG    ROCm agents detected: []
21:43:03-089741 DEBUG    ROCm version detected: None
21:43:03-091699 WARNING  HIP SDK is detected, but no Torch release for Windows available
21:43:03-092676 INFO     For ZLUDA support specify '--use-zluda'
21:43:03-093652 INFO     Using CPU-only torch
21:43:03-132899 ERROR    Error running git: None / status --porcelain
21:43:03-134851 DEBUG    Git output: fatal: detected dubious ownership in repository at 'E:/ia/automatic'
                         'E:/ia/automatic' is owned by:
                                 (inconvertible) (S-1-5-21-291463465-3360491487-3224188617-1001)
                         but the current user is:
                                 DESKTOP-6RE3Q6G/manotroll (S-1-5-21-2930017012-368641518-4144343359-1001)
                         To add an exception for this directory, call:

                                 git config --global --add safe.directory E:/ia/automatic
21:43:03-141723 WARNING  Deleted files: ["tal: detected dubious ownership in repository at 'E:/ia/automatic'",
                         ":/ia/automatic' is owned by:", 'inconvertible)
                         (S-1-5-21-291463465-3360491487-3224188617-1001)', 't the current user is:',
                         'ESKTOP-6RE3Q6G/manotroll (S-1-5-21-2930017012-368641518-4144343359-1001)', 'add an exception
                         for this directory, call:', 'it config --global --add safe.directory E:/ia/automatic']
21:43:03-239806 INFO     Verifying requirements
21:43:03-244694 INFO     Verifying packages
21:43:03-245670 INFO     Startup: standard
21:43:03-246647 INFO     Verifying submodules
21:43:03-540906 ERROR    Error running git: None / submodule
21:43:03-542859 DEBUG    Git output: fatal: detected dubious ownership in repository at 'E:/ia/automatic'
                         'E:/ia/automatic' is owned by:
                                 (inconvertible) (S-1-5-21-291463465-3360491487-3224188617-1001)
                         but the current user is:
                                 DESKTOP-6RE3Q6G/manotroll (S-1-5-21-2930017012-368641518-4144343359-1001)
                         To add an exception for this directory, call:

                                 git config --global --add safe.directory E:/ia/automatic
21:43:03-802667 ERROR    Error running git: None / submodule --quiet update --init --recursive
21:43:03-804171 DEBUG    Git output: fatal: detected dubious ownership in repository at 'E:/ia/automatic'
                         'E:/ia/automatic' is owned by:
                                 (inconvertible) (S-1-5-21-291463465-3360491487-3224188617-1001)
                         but the current user is:
                                 DESKTOP-6RE3Q6G/manotroll (S-1-5-21-2930017012-368641518-4144343359-1001)
                         To add an exception for this directory, call:

                                 git config --global --add safe.directory E:/ia/automatic
21:43:04-106220 ERROR    Error running git: None / submodule --quiet sync --recursive
21:43:04-108174 DEBUG    Git output: fatal: detected dubious ownership in repository at 'E:/ia/automatic'
                         'E:/ia/automatic' is owned by:
                                 (inconvertible) (S-1-5-21-291463465-3360491487-3224188617-1001)
                         but the current user is:
                                 DESKTOP-6RE3Q6G/manotroll (S-1-5-21-2930017012-368641518-4144343359-1001)
                         To add an exception for this directory, call:

                                 git config --global --add safe.directory E:/ia/automatic
21:43:04-116017 ERROR    Error updating submodule: fatal: detected dubious ownership in repository at 'E:/ia/automatic'
21:43:04-117970 ERROR    Error updating submodule: 'E:/ia/automatic' is owned by:
21:43:04-118946 ERROR    Error updating submodule:       (inconvertible)
                         (S-1-5-21-291463465-3360491487-3224188617-1001)
21:43:04-121909 ERROR    Error updating submodule: but the current user is:
21:43:04-123862 ERROR    Error updating submodule:       DESKTOP-6RE3Q6G/manotroll
                         (S-1-5-21-2930017012-368641518-4144343359-1001)
21:43:04-126790 ERROR    Error updating submodule: To add an exception for this directory, call:
21:43:04-127767 ERROR    Error updating submodule:
21:43:04-129749 ERROR    Error updating submodule:       git config --global --add safe.directory E:/ia/automatic
21:43:04-130730 DEBUG    Register paths
21:43:04-132683 ERROR    Required path not found: path=E:\ia\automatic\modules\k-diffusion\k_diffusion\sampling.py
                         item=k_diffusion
21:43:04-222932 DEBUG    Installed packages: 156
21:43:04-224885 DEBUG    Extensions all: ['Lora', 'sd-extension-chainner', 'sd-extension-system-info',
                         'sd-webui-agent-scheduler', 'sdnext-modernui', 'stable-diffusion-webui-rembg']
21:43:04-267027 ERROR    Error running git: extensions-builtin\Lora / pull origin None --rebase --force
21:43:04-268004 DEBUG    Git output: fatal: detected dubious ownership in repository at 'E:/ia/automatic'
                         'E:/ia/automatic' is owned by:
                                 (inconvertible) (S-1-5-21-291463465-3360491487-3224188617-1001)
                         but the current user is:
                                 DESKTOP-6RE3Q6G/manotroll (S-1-5-21-2930017012-368641518-4144343359-1001)
                         To add an exception for this directory, call:

                                 git config --global --add safe.directory E:/ia/automatic
21:43:04-409211 ERROR    Error running git: extensions-builtin\sd-extension-chainner / pull origin None --rebase
                         --force
21:43:04-412173 DEBUG    Git output: fatal: detected dubious ownership in repository at 'E:/ia/automatic'
                         'E:/ia/automatic' is owned by:
                                 (inconvertible) (S-1-5-21-291463465-3360491487-3224188617-1001)
                         but the current user is:
                                 DESKTOP-6RE3Q6G/manotroll (S-1-5-21-2930017012-368641518-4144343359-1001)
                         To add an exception for this directory, call:

                                 git config --global --add safe.directory E:/ia/automatic
21:43:04-551886 ERROR    Error running git: extensions-builtin\sd-extension-system-info / pull origin None --rebase
                         --force
21:43:04-554815 DEBUG    Git output: fatal: detected dubious ownership in repository at 'E:/ia/automatic'
                         'E:/ia/automatic' is owned by:
                                 (inconvertible) (S-1-5-21-291463465-3360491487-3224188617-1001)
                         but the current user is:
                                 DESKTOP-6RE3Q6G/manotroll (S-1-5-21-2930017012-368641518-4144343359-1001)
                         To add an exception for this directory, call:

                                 git config --global --add safe.directory E:/ia/automatic
21:43:04-692119 ERROR    Error running git: extensions-builtin\sd-webui-agent-scheduler / pull origin None --rebase
                         --force
21:43:04-694072 DEBUG    Git output: fatal: detected dubious ownership in repository at 'E:/ia/automatic'
                         'E:/ia/automatic' is owned by:
                                 (inconvertible) (S-1-5-21-291463465-3360491487-3224188617-1001)
                         but the current user is:
                                 DESKTOP-6RE3Q6G/manotroll (S-1-5-21-2930017012-368641518-4144343359-1001)
                         To add an exception for this directory, call:

                                 git config --global --add safe.directory E:/ia/automatic
21:43:04-826987 ERROR    Error running git: extensions-builtin\sdnext-modernui / pull origin None --rebase --force
21:43:04-828940 DEBUG    Git output: fatal: detected dubious ownership in repository at 'E:/ia/automatic'
                         'E:/ia/automatic' is owned by:
                                 (inconvertible) (S-1-5-21-291463465-3360491487-3224188617-1001)
                         but the current user is:
                                 DESKTOP-6RE3Q6G/manotroll (S-1-5-21-2930017012-368641518-4144343359-1001)
                         To add an exception for this directory, call:

                                 git config --global --add safe.directory E:/ia/automatic
21:43:04-959872 ERROR    Error running git: extensions-builtin\stable-diffusion-webui-rembg / pull origin None --rebase
                         --force
21:43:04-961829 DEBUG    Git output: fatal: detected dubious ownership in repository at 'E:/ia/automatic'
                         'E:/ia/automatic' is owned by:
                                 (inconvertible) (S-1-5-21-291463465-3360491487-3224188617-1001)
                         but the current user is:
                                 DESKTOP-6RE3Q6G/manotroll (S-1-5-21-2930017012-368641518-4144343359-1001)
                         To add an exception for this directory, call:

                                 git config --global --add safe.directory E:/ia/automatic
21:43:05-053041 DEBUG    Extensions all: []
21:43:05-054994 INFO     Extensions enabled: ['Lora', 'sd-extension-chainner', 'sd-extension-system-info',
                         'sd-webui-agent-scheduler', 'sdnext-modernui', 'stable-diffusion-webui-rembg']
21:43:05-056947 INFO     Verifying requirements
21:43:05-057923 INFO     Updating Wiki
21:43:05-097614 ERROR    Error running git: E:\ia\automatic\wiki / pull origin None --rebase --force
21:43:05-100094 DEBUG    Git output: fatal: detected dubious ownership in repository at 'E:/ia/automatic'
                         'E:/ia/automatic' is owned by:
                                 (inconvertible) (S-1-5-21-291463465-3360491487-3224188617-1001)
                         but the current user is:
                                 DESKTOP-6RE3Q6G/manotroll (S-1-5-21-2930017012-368641518-4144343359-1001)
                         To add an exception for this directory, call:

                                 git config --global --add safe.directory E:/ia/automatic
21:43:05-106490 WARNING  Setup complete with errors: 17
21:43:05-107466 WARNING  See log file for more details: E:\ia\automatic\sdnext.log
21:43:05-111905 DEBUG    Extension preload: {'extensions-builtin': 0.0, 'extensions': 0.0}
21:43:05-113858 DEBUG    Starting module: <module 'webui' from 'E:\\ia\\automatic\\webui.py'>
21:43:05-115811 INFO     Command line args: ['--experimental', '--debug', '--upgrade', '--update'] upgrade=True
                         experimental=True debug=True
21:43:05-118742 DEBUG    Env flags: []
┌───────────────────────────────────────── Traceback (most recent call last) ─────────────────────────────────────────┐
│ E:\ia\automatic\launch.py:265 in <module>                                                                           │
│                                                                                                                     │
│   264 if __name__ == "__main__":                                                                                    │
│ > 265     main()                                                                                                    │
│   266                                                                                                               │
│                                                                                                                     │
│ E:\ia\automatic\launch.py:242 in main                                                                               │
│                                                                                                                     │
│   241                                                                                                               │
│ > 242     uv, instance = start_server(immediate=True, server=None)                                                  │
│   243     while True:                                                                                               │
│                                                                                                                     │
│ E:\ia\automatic\launch.py:168 in start_server                                                                       │
│                                                                                                                     │
│   167     get_custom_args()                                                                                         │
│ > 168     module_spec.loader.exec_module(server)                                                                    │
│   169     uvicorn = None                                                                                            │
│ in exec_module:883                                                                                                  │
│ in _call_with_frames_removed:241                                                                                    │
│                                                                                                                     │
│ E:\ia\automatic\webui.py:11 in <module>                                                                             │
│                                                                                                                     │
│    10 from threading import Thread                                                                                  │
│ >  11 import modules.loader                                                                                         │
│    12 import torch # pylint: disable=wrong-import-order                                                             │
│                                                                                                                     │
│ E:\ia\automatic\modules\loader.py:14 in <module>                                                                    │
│                                                                                                                     │
│   13                                                                                                                │
│ > 14 import torch # pylint: disable=C0411                                                                           │
│   15 try:                                                                                                           │
│                                                                                                                     │
│ E:\ia\automatic\venv\lib\site-packages\torch\__init__.py:148 in <module>                                            │
│                                                                                                                     │
│    147                 err.strerror += f' Error loading "{dll}" or one of its dependencies.'                        │
│ >  148                 raise err                                                                                    │
│    149                                                                                                              │
└─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
OSError: [WinError 126] Não foi possível encontrar o módulo especificado. Error loading "E:\ia\automatic\venv\lib\site-packages\torch\lib\fbgemm.dll" or one of its dependencies.
```

### Backend

Diffusers

### UI

Standard

### Branch

Master

### Model

StableDiffusion 1.5

### Acknowledgements

- [X] I have read the above and searched for existing issues
- [X] I confirm that this is classified correctly and its not an extension issue

**Comment by brknsoul** [comment]
https://github.com/vladmandic/automatic/issues/3360#issuecomment-2272566647

**Comment by manotroll** [comment]
20:34:20-298224 INFO     Starting SD.Next
20:34:20-301222 INFO     Logger: file="E:\ia\automatic\sdnext.log" level=INFO size=28669 mode=append
20:34:20-302620 INFO     Python version=3.10.11 platform=Windows bin="E:\ia\automatic\venv\Scripts\python.exe"
                         venv="E:\ia\automatic\venv"
20:34:20-419812 INFO     Version: app=sd.next version=unknown branch=unknown ui=unknown
20:34:20-456489 ERROR    Error running git: None / rev-parse HEAD
20:34:20-591347 INFO     Latest published version: a874b27e50a343ac55148756a0a80eafb3a7f87f 2024-07-24T20:16:33Z
20:34:20-604886 INFO     Platform: arch=AMD64 cpu=AMD64 Family 25 Model 33 Stepping 0, AuthenticAMD system=Windows
                         release=Windows-10-10.0.22631-SP0 python=3.10.11
20:34:20-606885 INFO     HF cache folder: C:\Users\manotroll\.cache\huggingface\hub
20:34:20-612393 INFO     Python version=3.10.11 platform=Windows bin="E:\ia\automatic\venv\Scripts\python.exe"
                         venv="E:\ia\automatic\venv"
20:34:20-615395 INFO     AMD ROCm toolkit detected
20:34:20-648923 WARNING  HIP SDK is detected, but no Torch release for Windows available
20:34:20-649924 INFO     For ZLUDA support specify '--use-zluda'
20:34:20-651427 INFO     Using CPU-only torch
20:34:20-688960 ERROR    Error running git: None / status --porcelain
20:34:20-689960 WARNING  Deleted files: ["tal: detected dubious ownership in repository at 'E:/ia/automatic'",
                         ":/ia/automatic' is owned by:", 'inconvertible)
                         (S-1-5-21-291463465-3360491487-3224188617-1001)', 't the current user is:',
                         'ESKTOP-6RE3Q6G/manotroll (S-1-5-21-2930017012-368641518-4144343359-1001)', 'add an exception
                         for this directory, call:', 'it config --global --add safe.directory E:/ia/automatic']
20:34:20-784551 INFO     Verifying requirements
20:34:20-788554 INFO     Verifying packages
20:34:20-826094 ERROR    Error running git: None / log -1 --pretty=format:"%at"
20:34:20-827094 ERROR    Error getting local repository version: invalid literal for int() with base 10: "fatal:
                         detected dubious ownership in repository at 'E:/ia/automatic'\n'E:/ia/automatic' is owned
                         by:\n\t(inconvertible) (S-1-5-21-291463465-3360491487-3224188617-1001)\nbut the current user
                         is:\n\tDE
20:34:20-829094 ERROR    Uncaught exception occurred: type=<class 'OSError'> value=[Errno 22] Invalid argument
20:34:20-831094 ERROR    '  File "E:\\ia\\automatic\\launch.py", line 265, in <module>\n    main()\n'
20:34:20-831094 ERROR    '  File "E:\\ia\\automatic\\launch.py", line 223, in main\n    if
                         installer.check_timestamp():\n'
20:34:20-832439 ERROR    '  File "E:\\ia\\automatic\\installer.py", line 1195, in check_timestamp\n
                         log.debug(f\'Repository update time: {time.ctime(int(version_time))}\')\n'

**Comment by brknsoul** [comment]
open cmd and run `git config --global --add safe.directory *`
